Formula/dvc.rb

```diff
@@ -4,10 +4,9 @@ class Dvc < Formula
   desc "Git for data science projects"
   homepage "https://dvc.org"
   # TODO: Check if we can use unversioned `protobuf` at version bump
```

Comment:
We need to check this next time. Don't want to waste the long build run we've already done this time around.

Reply:
👍